src/log_database_proxy_model.cpp

```diff
@@ -244,6 +251,83 @@ bool LogDatabaseProxyModel::isExcludeValid() const
   return true;
 }
 
+//Locates the next index based on search criteria, VCM 25 April 2017
+//searchText_ - string from searchText, all upper case and trimmed spaces
+//index - currently selected item in messageList
+//increment - +1 = next||search(i.e. down), -1 = prev (i.e. up)
+int LogDatabaseProxyModel::getItemIndex(const QString searchText_, int index, int increment)
```

Comment:
The parameter names here don't match the names in the header; also, parameters should not end in an underscore.